src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -60,6 +62,30 @@
 logger = get_task_logger(__name__)
 
 
+def get_access_manual_webhook_inputs(
+    db: Session, privacy_request: PrivacyRequest
+) -> Tuple[Dict[str, List[Dict[str, Optional[Any]]]], bool]:
+    """Retrieves manually uploaded data for all AccessManualWebhooks and formats in a way
+    to match automatically retrieved data. Also returns if execution should proceed.
+
+    This data will be uploaded to the user as-is, without filtering.
+    """
+    manual_inputs: Dict[str, List[Dict[str, Optional[Any]]]] = {}
+
+    try:
+        for manual_webhook in AccessManualWebhook.get_enabled(db):
+            manual_inputs[manual_webhook.connection_config.key] = [
+                privacy_request.get_manual_webhook_input(manual_webhook)
+            ]
+    except (MissingManualWebhookData, ValidationError) as exc:
+        logger.info(exc)
+        privacy_request.status = PrivacyRequestStatus.requires_input
+        privacy_request.save(db)
+        return manual_inputs, False
+
+    return manual_inputs, True
```

Comment:
Normally the access results have keys in the format "dataset:collection", so here I am using the `manual_webhook` connectionconfig key instead, since there is no "dataset" or "collection" defined for the webhook results.   I think an overlap with existing keys is unlikely since they are mashups of dataset and collection names.

As an aside, since ConnectionConfigs and AccessManualWebhooks have a 1:1 relationship, I am not giving a separate identifier to the AccessManualWebhook, and using its ConnectionConfig key _everywhere_. So the question is, do we want them to be separate? We could store a separate AccessManualWebhook key and use that here, and on the UI it could be surfaced `connection_identifier`, like we use for saas configs. I didn't think we needed this extra identifier, but I can see an opposing argument.

Reply:
Discussed with Sean, seems okay for now to have this 1:1 mapping and use the connection_config key as the identifier for the entire webhook.